metrics/api/src/main/java/io/helidon/metrics/spi/MetersProvider.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.spi;
+
+import io.helidon.common.config.Config;
+import io.helidon.metrics.api.Meter;
+
+/**
+ * Behavior of providers of meters.
+ */
+public interface MetersProvider {
```

Comment:
Maybe this could use the `ConfiguredProvider`? This woudl add support for this provider into blueprints

Reply:
Hmm. 

IIUC this would allow users to use configuration to declare meters to be registered? 

I haven't worked with configured providers yet so I'll need to understand it better, I think.